src/senselab/audio/tasks/speech_to_text/api.py

```diff
@@ -22,16 +29,20 @@ def transcribe_audios(
         model (SenselabModel): The model used for transcription.
         language (Optional[Language]): The language of the audio (default is None).
         device (Optional[DeviceType]): The device to run the model on (default is None).
```

Comment:
device means "preferred device". Which is why we accept None in case the user doesn't have any preference and we can pick (based on performance). After some brainstorming we decided for device because it's shorter than preferred_device, but if that's so misleading we can change it everywhere. I have no strong preference. Feel free to open a PR changing this all over the package. I write a not about this

Reply:
Got it. I don't have a strong opinion on this either but I was confused by it. Is this behavior documented somewhere in senselab?